server/protocols/webadmin/webadmin-cassandra-data/src/main/java/org/apache/james/webadmin/service/CassandraMappingsSolveInconsistenciesTask.java

```diff
@@ -78,7 +79,8 @@ public String getType() {
     public Result run() {
         return cassandraMappingsSourcesDAO.removeAllData()
             .doOnError(e -> LOGGER.error("Error while cleaning up data in mappings sources projection table"))
-            .then(Mono.fromCallable(mappingsSourcesMigration::run))
+            .then(Mono.fromCallable(mappingsSourcesMigration::run)
+                .subscribeOn(Schedulers.elastic()))
```

Comment:
ReactorUtils.BLOCKING_CALL_WRAPPER please

Reply:
Even better, avoid blocking calls in reactor in the first place.

Here we can write:

```
    @Override
    public Result run() throws InterruptedException {
        cassandraMappingsSourcesDAO.removeAllData()
            .doOnError(e -> LOGGER.error("Error while cleaning up data in mappings sources projection table"))
            .block();
        return mappingsSourcesMigration.run();
    }
```